code/hud/hudsquadmsg.cpp

```diff
@@ -2098,6 +2097,18 @@ void hud_squadmsg_wing_command()
 		if ( !hud_squadmsg_is_target_order_valid((int)order_id, 0) )
 			MsgItems[Num_menu_items].active = 0;
 
+		// if no ship in the wing can depart then gray out the departure order
+		if (order_id == DEPART_ITEM) {
+			int active = 0;
+			for (int i = 0; i < wingp->current_count; i++) {
+				if (hud_squadmsg_ship_order_valid(wingp->ship_index[i], (int)order_id)) {
+					active = 1;
+					break;
+				}
+			}
+			MsgItems[Num_menu_items].active = active;
+		}
+
```

Comment:
Why is this check actually restricted to DEPART_ITEM?  Shouldn't wing orders always be unavailable if no ship in the wing can perform them?  And if so, are there any similar checks in other parts of the squadmsg code?

Reply:
I'm not sure why it's like that in the code, but indeed I reckon wing orders should be unavailable if no ship in the wing can perform them, which this PR seems to remedy in my tests. Also IIRC there's not really any similar checks for wings or all fighters.

Reply:
EDIT: Finally recall that adding this extra check is needed b/c warp ability can be set via flag and the code seemed to check for protecting and attacking orders already.